app/com/linkedin/drelephant/mapreduce/heuristics/MapperSpeedHeuristic.java

```diff
@@ -146,6 +147,8 @@ public HeuristicResult apply(MapReduceApplicationData data) {
     result.addResultDetail("Median task input size", FileUtils.byteCountToDisplaySize(medianSize));
     result.addResultDetail("Median task runtime", Statistics.readableTimespan(medianRuntimeMs));
     result.addResultDetail("Median task speed", FileUtils.byteCountToDisplaySize(medianSpeed) + "/s");
+    result.addResultDetail("Total input size in MB", totalInputByteSize*1.0/(1024*1024) + "");
```

Comment:
use standard constants for these

Reply:
This need to be changed overall for all heuristics. I will make the changes for this new addition for now.

Reply:
Will change this.

Reply:
Done by @mkumar1984